README.md

```diff
@@ -22,7 +22,10 @@ Install:
 
 Information about how to install for your platform can be found [here](https://docs.docker.com/install/).
 
-On Windows and MacOS you will need to install Docker Desktop.
+If you're not running Linux, good luck.
```

Comment:
Let's make this a little nicer, something like, installation is easiest on Linux where docker is supported. On MacOS and Windows 10 Pro Edition, Docker Desktop enables the functionality of docker, and on Windows 10 Home Edition, download Docker Toolbox.